contrib/test/ci/vars.yml

```diff
@@ -44,7 +44,7 @@ ssh_user: "{{ lookup('env','USER') }}"
 ssh_location: "{{ lookup('env','HOME') }}/.ssh/id_rsa"
 
 # variables for testing with the kata-containers runtime
-kata_version: "3.1.3"
+kata_version: "3.3.0"
```

Comment:
There's already a [PR](https://github.com/cri-o/cri-o/pull/7958) to test the new version of Kata.

Reply:
oh, thanks, I didn't spot this.